docs/deploy.md

```diff
@@ -104,14 +87,29 @@ authentication:
       password: user
 ```
 
-2. Run manager-api
+2. Run
 
 ```sh
-$ api/run.sh &
+$ cd ./output
+$ export ENV=local && exec ./manager-api
```

Comment:
ENV = local???

Reply:
confused too

Reply:
I checked code , in deploy ENV=local is not needed.